src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -195,13 +239,22 @@ export default {
                 this.paused(false);
                 this.unselectAllImages();
             } else {
+                this.paused(true);
                 this.imageUrl = this.getImageUrl(image);
                 this.time = this.getTime(image);
-                this.paused(true);
                 this.unselectAllImages();
                 image.selected = true;
             }
         },
+        getSelectedImage() {
+            let selected = this.imageHistory.find(image => image.selected);
```

Comment:
We have to track the `selectedImage` on `this`. We're doing an exhaustive array search triggered by `setInterval` which is running every 100ms, but the selectedImage won't change very often, so we don't need to search for it every time.

Reply:
tracking selectedImage on this now, updated whenever updateValues is called as that's when the image is changed.